releases/point-2022-4-release-note.md

```diff
@@ -0,0 +1,60 @@
+---
+description: This article lists new features, improvements, and bug fixes in SysKit Point version 2022.4.
+--- 
+
+# SysKit Point 2022.4
+
+[Schedule a demo](https://www.syskit.com/products/point/request-a-demo/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2022.4.0
+
+**Build number:** 5
+
+**Release date:** July 15, 2022
+
+## Get the Latest Version
+
+* **New to SysKit Point?** Get the latest SysKit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point).<br/>
+    Find all steps described in detail in the [deployment article](../installation/deploy-syskit-point.md).
+    
+* **Already using SysKit Point?** Upgrade to the latest SysKit Point version. <br/>
+[![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FUpdateFilesARM%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../installation/upgrade-syskit-point.md).
+
+* **Want to try out the Configuration Inventory module?** [Contact us](https://www.syskit.com/contact-us/).
+
+## Features
+
+* **Power Platform data sync can now be enabled for SysKit Point.** Collect Power BI data and utilize it for **Access Reports**.
+* Enabling Power BI data collection provides:
+    * List of all Power BI workspaces with **deeper insight into permissions and inventory**.
+    * Ability to explore all workspaces with a list of reports and databases, including information on **who created them, who the owner is, when it was last modified**, and more!
+    * Option to check which **users have access to Power BI workspaces and reports**, their level of access, and when they last used their licenses. 
```

Comment:
and their level of access i tu stati. ovaj dio sa licencama maknuti odavde. to ide u Power BI companion app, moram provjeriti da li to uopće spominjemo